index.js

```diff
@@ -210,9 +244,12 @@ class Tab extends EventEmitter {
     }
 
     setPosition (newPosition) {
+        if (newPosition === 0 || newPosition === undefined) {
```

Comment:
Most of the time we don't check parameters like this because it is endless and the user is supposed to check what is passed to the function. Is there a reason why we need to check `newPosition` here?

Reply:
I was checking for 0 because of 0 not being a valid position. I was also checking for `undefined` in order to require `newPosition` to be passed.

But, without these checks, it works fine and simply sets the tab position to the leftmost, so removing this check makes sense and works well.

I think this should be a comment, right? :smile:

Reply:
We assume programmers are passing valid parameters to functions. We won't anticipate every possible mistake in electron-tabs, so we don't need this. Not even a comment IMO.

Reply:
Okay, I will remove this.

I originally pressed the 'review' button not the 'comment' button, that is what I meant by it should have been a comment.

Reply:
> I originally pressed the 'review' button not the 'comment' button, that is what I meant by it should have been a comment.

Oh, well. I don't see the difference anyway 😃